pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/SchemasResourceBase.java

```diff
@@ -306,6 +308,29 @@ private static void handleGetAllSchemasResponse(AsyncResponse response, List<Sch
         }
     }
 
+    private void checkTopicExists(AsyncResponse response) {
+        pulsar().getNamespaceService().getListOfTopics(topicName.getNamespaceObject(),
```

Comment:
I think it's better to do not to add this check right now, if we have many topics under the namespace, this will bring more workload to Zookeeper.

Reply:
> I think it's better to do not to add this check right now, if we have many topics under the namespace, this will bring more workload to Zookeeper.

Get! I will close this PR

Reply:
Get! I will close this PR

Reply:
Thanks @yuruguo